recipes/pyriemann/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "pyriemann" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1feed8f72d94414bdc9ca4485333711a3f91f9742ae8794874a7399f4166758a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - scikit-learn >=0.17
+    - pandas
+    - joblib
+    - mne-base
+    - matplotlib-base
+    - seaborn
+
+test:
+  imports:
+    - pyriemann
+    - pyriemann.classification
+    - pyriemann.clustering
+    - pyriemann.embedding
+    - pyriemann.estimation
+    - pyriemann.spatialfilters
+    - pyriemann.stats
+    - pyriemann.tangentspace
+    - pyriemann.utils
+    - pyriemann.utils.covariance
+    - pyriemann.utils.distance
+    - pyriemann.utils.mean
+    - pyriemann.utils.viz
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://pyriemann.readthedocs.io/en/latest/
+  summary: pyRiemann is a Python machine learning library based on scikit-learn API. It provides a high-level interface for classification and manipulation of multivariate signal through Riemannian Geometry of covariance matrices.
```

This summary is almost as long as the description. Let's use the website summary:

```suggestion
  summary: pyRiemann: Biosignals classification with Riemannian Geometry
```